tests/scene/test_code_edit.h

```diff
@@ -2943,6 +2943,11 @@ TEST_CASE("[SceneTree][CodeEdit] completion") {
 			code_edit->set_code_completion_enabled(true);
 			CHECK(code_edit->get_code_completion_selected_index() == -1);
 
+			// `NOTIFICATION_THEME_CHANGED` is called deferred from `NOTIFICATION_ENTER_TREE`,
+			// but it won't have been called yet here, and `update_code_completion_options` requires access
+			// to the theme's font size, so we need to force an update here.
+			code_edit->notification(Control::NOTIFICATION_THEME_CHANGED);
```

Comment:
So https://github.com/godotengine/godot/blob/29ba1fc1be653858a936593fc373d23a1ab548da/scene/gui/code_edit.cpp#L2748 seems to depend on having `font` initialized, but currently with https://github.com/godotengine/godot/pull/62845 theme items aren't initialized *immediately* when entering the tree, instead the call is deferred because `add_child_notify` is called after `NOTIFICATION_ENTER_TREE`, but `NOTIFICATION_THEME_CHANGED` shouldn't be called until after `add_child_notify` because `add_child_notify` assigns what Control/Window the added child should inherit theme items from.

@Paulb23 You're the expert on `CodeEdit`, if you had a second to give your thoughts on this it'd be greatly appreciated 🙂

Reply:
If this is only happening in the unit tests, something like this is kind of expected as it's not really processing anything normally. Instead the message queue has to be forced with `MessageQueue::get_singleton()->flush();` calls. Most of these calls however are hidden in the test macros, so in this case manually calling the notifications is okay, or some kind of macro to hide it would also work.